rosa_cluster_admin/rosa_monitoring/rosa-managing-metrics.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="rosa-managing-metrics"]
+= Managing metrics
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-managing-metrics
+
+toc::[]
+
+This document provides an overview about how {product-title} metrics are collected, queried and visualized.
+
+// Understanding metrics
+include::modules/osd-monitoring-understanding-metrics.adoc[leveloffset=+1]
+
+// Setting up metrics collection for user-defined projects
+include::modules/osd-monitoring-setting-up-metrics-collection-for-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-deploying-a-sample-service.adoc[leveloffset=+2]
+include::modules/osd-monitoring-specifying-how-a-service-is-monitored.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the link:https://github.com/openshift/prometheus-operator/blob/release-4.7/Documentation/api.md[Prometheus Operator API documentation] for more information on `ServiceMonitor` and `PodMonitor` resources.
+* xref:../rosa_monitoring/rosa-accessing-monitoring-for-user-defined-projects.adoc#rosa-accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects].
+
+// Querying metrics
+include::modules/osd-monitoring-querying-metrics.adoc[leveloffset=+1]
+include::modules/osd-monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+include::modules/osd-monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
+* See the xref:../rosa_monitoring/rosa-managing-metrics.adoc#querying-metrics-for-user-defined-projects-as-a-developer_rosa-managing-metrics[Querying metrics for user-defined projects as a developer] for details on accessing non-cluster metrics as a developer or a privileged user
+// TODO: The above additional resource points to the same module that it's an additional resource of
+
+include::modules/osd-monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the xref:../rosa_monitoring/rosa-managing-metrics.adoc#querying-metrics_rosa-managing-metrics[Querying metrics] section on using the PromQL interface
+* xref:../rosa_monitoring/rosa-troubleshooting-monitoring-issues.adoc#rosa-troubleshooting-monitoring-issues[Troubleshooting monitoring issues]
+
+[id="managing-metrics-next-steps"]
+== Next steps
+* xref:../rosa_monitoring/rosa-managing-alerts.adoc#rosa-managing-alerts[Alerts]
```

Comment:
Would it make more sense to link to the same next steps as the alerts page instead of the empty alerts page?